model/legacy-private/src/main/java/org/keycloak/services/scheduled/ClusterAwareScheduledTaskRunner.java

```diff
@@ -44,8 +44,6 @@ public ClusterAwareScheduledTaskRunner(KeycloakSessionFactory sessionFactory, Sc
 
     @Override
     protected void runTask(final KeycloakSession session) {
-        session.getTransactionManager().begin();
```

Comment:
How is the transaction started in this case?

Reply:
It is started in the parent: `ScheduledTaskRunner.run` which calls `runTask`